wled00/util.cpp

```diff
@@ -504,7 +504,7 @@ um_data_t* simulateSound(uint8_t simulationId)
 
 
 void enumerateLedmaps() {
-  ledMaps = 1;
+  ledMaps = 0;
```

Comment:
As stated above, ledmap 0 always *exists*. If there is no file "ledmap.json" it means no ledmap is used, use default (ever increasing) LED order (AKA clear current ledmap).